atomic_reactor/plugins/pre_check_and_set_platforms.py

```diff
@@ -38,6 +39,25 @@ def __init__(self, workflow, koji_target=None):
         super(CheckAndSetPlatformsPlugin, self).__init__(workflow)
         self.koji_target = koji_target
 
+    def _limit_platforms(self, platforms: List[str]) -> List[str]:
+        """Limit platforms in a specific range by platforms config.
+
+        :param platforms: a list of platforms to be filtered.
+        :type platforms: list[str]
+        :return: the limited platforms.
+        :rtype: list[str]
+        """
+        final_platforms = set(platforms)
+        source_config = self.workflow.source.config
+        only_platforms = set(source_config.only_platforms)
+        excluded_platforms = set(source_config.excluded_platforms)
+
+        if only_platforms:
+            if only_platforms == excluded_platforms:
+                self.log.warning('only and not platforms are the same: %r', only_platforms)
```

Comment:
I realize that this was copied over verbatim, but I'd like to see it clarified, something like

> 'only' and 'not' platform lists are identical

It's pretty cryptic as it stands, unless you *already* know what it's talking about